aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -1800,6 +1817,20 @@ func resourceAviatrixSpokeGatewayUpdate(d *schema.ResourceData, meta interface{}
 		}
 	}
 
+	if d.HasChange("enable_auto_advertise_s2c_cidrs") {
+		if d.Get("enable_auto_advertise_s2c_cidrs").(bool) {
+			err := client.EnableAutoAdvertiseS2CCidrs(gateway)
+			if err != nil {
+				return fmt.Errorf("could not enable auto advertise s2c cidrs during spoke gateway update")
```

Comment:
Does the API error need to be added to the message?

Reply:
Yes! Good catch, I need to make a linter rule to catch this or something.